posneg.c

```diff
@@ -0,0 +1,19 @@
+/* Matthew Danielson 
+*  6/29/16
+*  posneg.c
+*  Evalutes whether or not an integer is positive or negative
+*/
+#include <stdio.h>
+
+int main(){
+	int x;
```

Comment:
Your program was supposed to handle floats - once again, follow directions more carefully.